-------------------------------------------------------------------------------
Test set: ForoAluraApi.ForoApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.992 s <<< FAILURE! - in ForoAluraApi.ForoApplicationTests
contextLoads  Time elapsed: 0.004 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext for [WebMergedContextConfiguration@4ac0ed65 testClass = ForoAluraApi.ForoApplicationTests, locations = [], classes = [ForoAluraApi.ForoApplication], contextInitializerClasses = [], activeProfiles = [], propertySourceLocations = [], propertySourceProperties = ["org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true"], contextCustomizers = [org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@18df8434, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@1e0b4072, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@5ab956d7, org.springframework.boot.test.autoconfigure.actuate.observability.ObservabilityContextCustomizerFactory$DisableObservabilityContextCustomizer@9da1, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@793f29ff, org.springframework.boot.test.context.SpringBootTestAnnotation@1b8b1ee9], resourceBasePath = "src/main/webapp", contextLoader = org.springframework.boot.test.context.SpringBootContextLoader, parent = null]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Collection 'ForoAluraApi.domain.topicos.Topico.respuestas' is 'mappedBy' a property named 'topicos' which does not exist in the target entity 'ForoAluraApi.domain.respuestas.Respuesta'
Caused by: org.hibernate.AnnotationException: Collection 'ForoAluraApi.domain.topicos.Topico.respuestas' is 'mappedBy' a property named 'topicos' which does not exist in the target entity 'ForoAluraApi.domain.respuestas.Respuesta'

